compiler/circle-mpqsolver/src/bisection/TestHelper.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
DRAFT shows that this file is shared, but has moved into `bisection` folder.
Is this OK?

Reply:
Currently the only implemented method is bisection. So for now it's ok.

Reply:
> So for now it's ok.

If you plan to move this file, it would be better to put it where it should by as you have the full draft.

Reply:
> If you plan to move this file, it would be better to put it where it should be as you have the full draft.

I see. I expect all modules needed for `bisection` algorithm will placed into `bisection` folder, so unit-tests will also be placed into `bisection` folder. Someday later the situation may change, then we can make some refactoring.

Reply:
> then we can make some refactoring.

OK got it